### PR TITLE
tck build image & wheel package

### LINKED ISSUE #27: Run the TCK with cloudstateio [proxy|tck]-docker images
The current Dockerfile for the shopping cart does not work to run the TCK properly.

### PR DESCRIPTION
**Pull request by marcellanz**
This PR should fix #27 and prepare part for #28.
- create a wheel package for distribution
- having the TCK docker image this wheel package installed
- added the TCK run script.
- added license and copyright headers.

This PR runs the against the TCK docker images of the main project.

### CODE REVIEW

**Comment by marcellanz** [comment]
@pvlugter I used an image repository of my own for the docker image we create for the python tck docker image in this PR. Doing this similar for the Go Support and I think [Adriano](@sleipnir) is doing the same for Dart and Kotlin.

While it's fine for me for now, and the images are quite small (at least for Go :-) ) you might want to have them in the cloudstate(io) docker repo or the like at some point? This is also not urgent, as these images are not used by users usually.
 
- gcr.io/mrcllnz/cloudstate-python-tck
- gcr.io/mrcllnz/cloudstate-go-tck-dev:latest
- gcr.io/mrcllnz/cloudstate-go-tck:latest

File: cloudstate/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""
+Copyright 2020 Lightbend Inc.
+Licensed under the Apache License, Version 2.0.
+"""
+
+from .version import __version__
```
Comment (marcellanz) [inline cloudstate/__init__.py:6]: note: this is needed so that setup.py detects this directory as a module directory.

File: setup.py [hunk 2]
```diff
@@ -14,4 +19,4 @@
       description='Cloudstate Python Support Library',
       packages=find_packages(exclude=['tests', 'shoppingcart']),
       long_description=open('README.md').read(),
-      zip_safe=False)
\ No newline at end of file
+      zip_safe=False)
```
Comment (marcellanz) [inline setup.py:22]: note: I see many complaints by my IDE (PyCharm) about missing newlines on the last line (PEP 8: W292 no newline at end of file), so corrected this here. This is needed or good practice by other languages or filetypes, we might adhere to it too.

**Comment by sleipnir** [comment]
LGTM. We just have to make sure that it runs after the build and after pushing the image to the repository. Which is why I would like an image that does not depend on our build because I see the shoppin-cart image as something independent. That serves as an example for the user and things like cloudstate CLI. So I find the previous docker image simpler and serving a purpose other than TCK.
But I think for now we can go on like this

**Comment by pvlugter** [comment]
> @pvlugter I used an image repository of my own for the docker image we create for the python tck docker image in this PR. Doing this similar for the Go Support and I think [Adriano](@sleipnir) is doing the same for Dart and Kotlin.
> 
> While it's fine for me for now, and the images are quite small (at least for Go :-) ) you might want to have them in the cloudstate(io) docker repo or the like at some point? This is also not urgent, as these images are not used by users usually.
> 
> * gcr.io/mrcllnz/cloudstate-python-tck
> * gcr.io/mrcllnz/cloudstate-go-tck-dev:latest
> * gcr.io/mrcllnz/cloudstate-go-tck:latest

We can certainly have these published under cloudstateio on docker hub. I can create repositories and teams. Let me know docker hub usernames to give access.

**Comment by marcellanz** [comment]
@pvlugter Thank you. my docker hub handle is: `mrcllnz`

**Comment by pvlugter** [comment]
@marcellanz added you to python and go teams under cloudstateio org in docker hub, with access to these newly created repositories:

```
cloudstateio/cloudstate-python-tck
cloudstateio/cloudstate-go-tck
cloudstateio/cloudstate-go-tck-dev
```

**Comment by marcellanz** [comment]
@pvlugter thank you. I have pushed the relevant go-tck and python-tck images over to docker hub. [TCK-nightly](https://observablehq.com/@marcellanz/cloudstate-go-tck-verification) is updated to reflect that. 

I have one thing left, I don't need the repository`cloudstateio/cloudstate-go-tck-dev`, I think we can delete it, I used it on gcr.io but can easily use a tag in the `cloudstateio/cloudstate-go-tck` repository we now have.

**Comment by pvlugter** [comment]
Okay, I've deleted `cloudstateio/cloudstate-go-tck-dev`

**Pull request merged**